cmd/util.go

```diff
@@ -36,6 +37,11 @@ func debugPrint() {
 	}
 }
 
+func isInContainer() bool {
+	host, port := os.Getenv("KUBERNETES_SERVICE_HOST"), os.Getenv("KUBERNETES_SERVICE_PORT")
```

Comment:
It would be a good idea to make those strings ("KUBERNETES_SERVICE_HOST", "KUBERNETES_SERVICE_PORT") as `const`

It would be good to use `os.LookupEnv` instead of using `os.Getenv` and checking the length